modules/centralized-logging/variables.tf

```diff
@@ -69,3 +69,9 @@ variable "kms_key_protection_level" {
   type        = string
   default     = "HSM"
 }
+
+variable "data_access_logs_enabled" {
```

Comment:
I thought data logging was in a different PR, so maybe something go accidentally mixed up.